# try-invoke
Attempts to invoke the given function, returning either the result or the caught error object.
